orchagent/neighorch.cpp

```diff
@@ -768,13 +815,6 @@ bool NeighOrch::removeNeighbor(const NeighborEntry &neighborEntry, bool disable)
         return true;
     }
 
-    if (m_syncdNextHops.find(nexthop) != m_syncdNextHops.end() && m_syncdNextHops[nexthop].ref_count > 0)
```

Comment:
Why is this removed? This flow is used in dual-tor scenario. Suggest to put it back.

Reply:
@TACappleman  Tom made these changes and can address your question.

Reply:
This change doesn't seem to be ours (or at least I can't find any commit where we did it). I suspect it got caught up in a rebase somewhere, so should probably be undone.

Reply:
This code has been recovered.